.gitignore

```diff
@@ -28,3 +28,4 @@ out.txt
 
 /manifests/**/*.yml.lock
 
+opensearch-*.tar.gz
```

Comment:
Created an issue to track https://github.com/opensearch-project/opensearch-build/issues/1109

Reply:
Since you’re still working on changes, I think you should fix this one before merging, it’s a 1-line change and gets rid of the addition to .gitignore.

Reply:
Sure. Will fix as part of this PR

Reply:
Researched a bit for this and found that the tar file is not created by integ test script. I checked out a clean workspace, run build script, verify no tar file in root dir, run **assemble** script and then see the tar file in root dir.

Then dig into the assemble workflow and find the issue at this line https://github.com/opensearch-project/opensearch-build/blob/main/src/assemble_workflow/dist.py#L76 where work dir is not passed in.

Thus I would suggest using the bug issue above to track this issue separately. (I will need to update that bug description to reflect latest situation). Then the remaining comments about the dir in this PR may no longer apply. (I verified that once the tar is manually deleted post assemble, running integ test script will not create the tar)